src/api/v2/updates.rs

```diff
@@ -0,0 +1,230 @@
+use std::collections::HashMap;
+use std::str::FromStr;
+
+use crate::db::types::BcdUpdateEventType;
+use crate::db::v2::bcd_updates::get_bcd_updates_paginated;
+use crate::db::v2::model::{Event, Status};
+use crate::{api::error::ApiError, db::Pool};
+use actix_identity::Identity;
+use actix_web::{web, HttpRequest, HttpResponse};
+use chrono::NaiveDate;
+use itertools::Itertools;
+use serde::{Deserialize, Serialize};
+
+fn array_like<'de, D>(deserializer: D) -> Result<Option<Vec<String>>, D::Error>
+where
+    D: serde::Deserializer<'de>,
+{
+    let s: Option<String> = Option::deserialize(deserializer)?;
+    if let Some(res) = s {
+        let collected: Vec<String> = res
+            .split(',')
+            .map(|val| String::from_str(val).unwrap())
+            .collect();
+        return Ok(Some(collected));
+    }
+    Ok(None)
+}
+#[derive(Debug, Serialize, Deserialize)]
+pub enum AscOrDesc {
+    #[serde(alias = "asc")]
+    Asc,
+    #[serde(alias = "desc")]
+    Desc,
+}
+
+#[derive(Deserialize, Serialize)]
+pub struct BcdUpdatesQueryParams {
+    #[serde(default, deserialize_with = "array_like")]
+    pub browsers: Option<Vec<String>>,
+    #[serde(default, deserialize_with = "array_like")]
+    pub category: Option<Vec<String>>,
+    pub live_since: Option<NaiveDate>,
```

Comment:
I'd drop this param, as we're not using it. If you put it in a single commit, you can always bring it back.